### PR TITLE
feat: add hotspot page

### PR DESCRIPTION
**Pull request by nicolorevelli**
Add Hotspot page

Card: https://trello.com/c/ginBsBCe/139-hotspot-ui

### CODE REVIEW

File: public/i18n/en/translation.json [hunk 1]
```diff
@@ -483,6 +491,62 @@
...
+        "configurtion_network_address": "Network address",
+        "configurtion_network_address_helper": "Lorem ipsum, or lipsum as it is sometimes known",
+        "configurtion_network_address_placeholder": "192.168.0.0/24",
+        "configurtion_dhcp_range_start": "DHCP range start",
+        "configurtion_dhcp_range_start_helper": "Lorem ipsum, or lipsum as it is sometimes known",
+        "configurtion_dhcp_range_start_placeholder": "192.168.0.2",
+        "configurtion_dhcp_range_end": "DHCP range end",
+        "configurtion_dhcp_range_end_helper": "Lorem ipsum, or lipsum as it is sometimes known",
+        "configurtion_dhcp_range_end_placeholder": "192.168.0.254",
+        "configurtion_max_client_allowed": "Max clients allowed:",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:536]:
```suggestion
        "configurtion_max_client_allowed": "Max allowed clients:",
```

File: public/i18n/en/translation.json [hunk 2]
```diff
@@ -483,6 +493,65 @@
       "route_onlink": "On-link",
       "route_onlink_description": "When enabled, gateway is on-link even if the gateway does not match any interface prefix"
     },
+    "hotspot": {
+      "title": "Hotspot",
+      "description": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:498]:
```suggestion
      "description": "Connect the firewall to a remote hotspot manager instance.",
```

File: public/i18n/en/translation.json [hunk 3]
```diff
@@ -483,6 +493,65 @@
...
+      "tabs": {
+        "status": "Status",
+        "settings": "Settings"
+      },
+      "status": {
+        "no_hotspot_configuration_found": "No Hotspot configuration found",
+        "no_hotspot_configuration_found_description": "Authenticate in \"Settings\" to edit hotspot configuration",
+        "no_hotspot_configuration_found_button": "Go to settings",
+        "empty_sessions": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:507]:
```suggestion
        "empty_sessions": "No active sessions",
```

File: public/i18n/en/translation.json [hunk 4]
```diff
@@ -483,6 +493,65 @@
...
+        "empty_sessions_description": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:508]:
```suggestion
        "empty_sessions_description": "The sessions of clients connected to this unit will appear here.",
```

File: public/i18n/en/translation.json [hunk 5]
```diff
@@ -483,6 +493,65 @@
...
+        "downloaded": "Downloaded",
+        "uploaded": "Uploaded"
+      },
+      "settings": {
+        "login": "Login",
+        "login_description": "Authenticate to edit hotspot configuration or unregister your hotspot.",
+        "login_hostname": "Host name",
+        "login_username": "Username",
+        "login_password": "Password",
+        "login_helper_username": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:524]:
```suggestion
        "login_helper_username": "Enter the email address associated with your Flashstart service subscription.",
```

File: public/i18n/en/translation.json [hunk 6]
```diff
@@ -72,7 +76,13 @@
...
     "cannot_retrieve_protocols": "Cannot retrieve protocols",
     "cannot_retrieve_zones": "Cannot retrieve zones",
-    "cannot_retrieve_wan_interfaces": "Cannot retrieve WAN interfaces"
+    "cannot_retrieve_wan_interfaces": "Cannot retrieve WAN interfaces",
+    "cannot_login_hotspot": "Cannot login",
+    "cannot_login_hotspot_description": "Login failed",
+    "cannot_unregister_hotspot": "Cannot unregister",
+    "cannot_load_hotspot_configuration": "Cannot load hotspot configuration",
+    "empty_network_device": "No \"Network device\" found.",
+    "empty_network_device_description": "There be at latest an unassigned device or with role \"hotspot\""
```
Comment (andre8244) [inline public/i18n/en/translation.json:85]:
```suggestion
    "empty_network_device_description": "There must be at latest an unassigned device or with role 'hotspot'"
```

File: public/i18n/en/translation.json [hunk 7]
```diff
@@ -72,7 +76,13 @@
     "cannot_disable_port_forward": "Cannot disable port forward",
     "cannot_retrieve_protocols": "Cannot retrieve protocols",
     "cannot_retrieve_zones": "Cannot retrieve zones",
-    "cannot_retrieve_wan_interfaces": "Cannot retrieve WAN interfaces"
+    "cannot_retrieve_wan_interfaces": "Cannot retrieve WAN interfaces",
+    "cannot_login_hotspot": "Cannot login",
+    "cannot_login_hotspot_description": "Login failed",
+    "cannot_unregister_hotspot": "Cannot unregister",
+    "cannot_load_hotspot_configuration": "Cannot load hotspot configuration",
+    "empty_network_device": "No \"Network device\" found.",
```
Comment (andre8244) [inline public/i18n/en/translation.json:84]:
```suggestion
    "empty_network_device": "No network device found",
```

File: public/i18n/en/translation.json [hunk 8]
```diff
@@ -483,6 +493,65 @@
...
     },
+    "hotspot": {
+      "title": "Hotspot",
+      "description": "Connect the firewall to a remote hotspot manager instance.",
+      "tabs": {
+        "status": "Status",
+        "settings": "Settings"
+      },
+      "status": {
+        "no_hotspot_configuration_found": "No Hotspot configuration found",
```
Comment (andre8244) [inline public/i18n/en/translation.json:504]:
```suggestion
        "no_hotspot_configuration_found": "No hotspot configuration found",
```

File: public/i18n/en/translation.json [hunk 9]
```diff
@@ -483,6 +493,65 @@
...
+        "no_hotspot_configuration_found_description": "Authenticate in \"Settings\" to edit hotspot configuration",
```
Comment (andre8244) [inline public/i18n/en/translation.json:505]:
```suggestion
        "no_hotspot_configuration_found_description": "Authenticate in Settings tab to configure hotspot",
```

File: public/i18n/en/translation.json [hunk 10]
```diff
@@ -483,6 +493,65 @@
...
+        "no_hotspot_configuration_found_button": "Go to settings",
```
Comment (andre8244) [inline public/i18n/en/translation.json:506]:
```suggestion
        "no_hotspot_configuration_found_button": "Go to Settings",
```

File: public/i18n/en/translation.json [hunk 11]
```diff
@@ -483,6 +493,65 @@
...
+        "configurtion_dhcp_range_end": "DHCP range end",
+        "configurtion_dhcp_range_end_helper": "Lorem ipsum, or lipsum as it is sometimes known",
+        "configurtion_dhcp_range_end_placeholder": "192.168.0.254",
+        "configurtion_max_client_allowed": "Max allowed clients:",
+        "configurtion_max_client_allowed_placeholder": "253",
+        "configurtion_parent_hotspot": "Parent hotspot",
+        "configurtion_parent_hotspot_placeholder": "samplename-office",
+        "configurtion_network_device": "Network device",
+        "configurtion_network_device_helper": "Lorem ipsum, or lipsum as it is sometimes known",
+        "configurtion_network_device_placeholder": "eth1",
```
Comment (andre8244) [inline public/i18n/en/translation.json:545]:
```suggestion
        "configurtion_network_device_placeholder": "Choose device",
```

File: public/i18n/en/translation.json [hunk 12]
```diff
@@ -483,6 +493,65 @@
...
+        "configurtion_use_default_dhcp_range": "Use default DHCP range",
+        "configurtion_save_info": "Save & commit",
```
Comment (andre8244) [inline public/i18n/en/translation.json:547]:
```suggestion
        "configurtion_save_info": "Save and commit",
```

File: public/i18n/en/translation.json [hunk 13]
```diff
@@ -483,6 +493,65 @@
...
+        "configurtion_save_info_description": "All pending changes to the firewall and network configuration will be committed.",
+        "unregister": "Unregister",
+        "unregister_unit": "Unregister unit",
+        "unregister_unit_modal_title": "Unregister Hotspot unit",
```
Comment (andre8244) [inline public/i18n/en/translation.json:551]:
```suggestion
        "unregister_unit_modal_title": "Unregister hotspot unit",
```

File: public/i18n/en/translation.json [hunk 14]
```diff
@@ -483,6 +493,65 @@
...
+        "unregister_unit_modal_description": "The Hotspot unit will be detached from the remote Hotspot manager. Local firewall and network configuration will not be changed."
```
Comment (andre8244) [inline public/i18n/en/translation.json:552]:
```suggestion
        "unregister_unit_modal_description": "The hotspot unit will be detached from the remote hotspot manager. Local firewall and network configurations will not be changed."
```

File: public/i18n/en/translation.json [hunk 15]
```diff
@@ -483,6 +493,65 @@
...
+        "configurtion_max_client_allowed": "Max allowed clients:",
+        "configurtion_max_client_allowed_placeholder": "253",
+        "configurtion_parent_hotspot": "Parent hotspot",
+        "configurtion_parent_hotspot_placeholder": "samplename-office",
+        "configurtion_network_device": "Network device",
+        "configurtion_network_device_helper": "Lorem ipsum, or lipsum as it is sometimes known",
+        "configurtion_network_device_placeholder": "eth1",
+        "configurtion_use_default_dhcp_range": "Use default DHCP range",
+        "configurtion_save_info": "Save & commit",
+        "configurtion_save_info_description": "All pending changes to the firewall and network configuration will be committed.",
```
Comment (andre8244) [inline public/i18n/en/translation.json:548]:
```suggestion
        "configurtion_save_info_description": "All pending changes to firewall and network configurations will be committed.",
```

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 16]
```diff
@@ -0,0 +1,695 @@
...
+
+  // Retrieve parent hotspot
+  try {
+    let getParentHotspot = await ubusCall('ns.dedalo', 'list-parents', {})
+    if (
+      getParentHotspot &&
+      getParentHotspot.data &&
+      getParentHotspot.data.parents &&
+      getParentHotspot.data.parents.length
+    ) {
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:129]: This `if` statement can be simplified to:
```
if (getParentHotspot?.data?.parents?.length) {
  ...
```

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 17]
```diff
@@ -0,0 +1,695 @@
...
+  getConfiguration()
+})
+
+async function getFirewallData() {
+  loading.value = true
+  errorLoadingData.value = { ...objError }
+
+  // Retrieve parent hotspot
+  try {
+    let getParentHotspot = await ubusCall('ns.dedalo', 'list-parents', {})
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:123]: `getParentHotspot` sounds like the name of a function, let's rename this variable to something like `listParentsResponse` or simply `res`

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 18]
```diff
@@ -0,0 +1,695 @@
...
+        label: item.name
+      }))
+      configurationForm.value.parentHotspot = getParentHotspot.data.parents[0].id
+    }
+  } catch (exception: any) {
+    errorLoadingData.value.notificationTitle = t('error.cannot_retrieve_parent_hotspot')
+    errorLoadingData.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  } finally {
+    try {
+      let getNetworkDevice = await ubusCall('ns.dedalo', 'list-devices', {})
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:142]: If there are no dependencies between `list-parents` and `list-devices` we can invoke them at the same time. We can split `getFirewallData` function and invoke the two functions inside `onMouted`

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 19]
```diff
@@ -0,0 +1,695 @@
...
+      if (
+        getNetworkDevice &&
+        getNetworkDevice.data &&
+        getNetworkDevice.data.devices &&
+        getNetworkDevice.data.devices.length
+      ) {
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:148]: See above comments

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 20]
```diff
@@ -0,0 +1,695 @@
...
+
+async function getConfiguration() {
+  try {
+    let getDataConfiguration = await ubusCall('ns.dedalo', 'get-configuration', {})
+    if (
+      getDataConfiguration &&
+      getDataConfiguration.data &&
+      getDataConfiguration.data.configuration &&
+      getDataConfiguration.data.configuration.connected
+    ) {
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:172]: See comments above

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 21]
```diff
@@ -0,0 +1,695 @@
...
+    let { valid, errMessage } = validateRequired(loginForm.value.hostname)
+    if (!valid) {
+      error.value.hostname = t(errMessage as string)
+      isValidationOk = false
+    }
+  }
+
+  if (!isValidationOk) {
+    focusElement(hostnameRef)
+    isFocusInput = true
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:207]: `focusElement(...)` and `isFocusInput = true` can be moved under the previous `isValidationOk = false`

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 22]
```diff
@@ -0,0 +1,695 @@
...
+
+  if (!loginForm.value.username) {
+    let { valid, errMessage } = validateRequired(loginForm.value.username)
+    if (!valid) {
+      error.value.username = t(errMessage as string)
+      isValidationOk = false
+    }
+  }
+
+  if (!isValidationOk && !isFocusInput) {
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:218]: See comment above

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 23]
```diff
@@ -0,0 +1,695 @@
...
+
+  if (!loginForm.value.password) {
+    let { valid, errMessage } = validateRequired(loginForm.value.password)
+    if (!valid) {
+      error.value.password = t(errMessage as string)
+      isValidationOk = false
+    }
+  }
+
+  if (!isValidationOk && !isFocusInput) focusElement(passwordRef)
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:231]: See comment above

File: src/components/standalone/hotspot/SettingsContent.vue [hunk 24]
```diff
@@ -0,0 +1,695 @@
...
+  if (!configurationForm.value.unitName) {
+    let { valid, errMessage } = validateRequired(configurationForm.value.unitName)
+    if (!valid) {
+      error.value.unitName = t(errMessage as string)
+      isValidationOk = false
+    }
+  }
+
+  if (!isValidationOk) {
+    focusElement(unitNameRef)
```
Comment (andre8244) [inline src/components/standalone/hotspot/SettingsContent.vue:280]: See comments above

File: src/components/standalone/hotspot/StatusContent.vue [hunk 25]
```diff
@@ -0,0 +1,212 @@
...
+
+onMounted(async () => {
+  await getConfiguration()
+  await loadListSessions()
+  loading.value = false
+})
+
+async function getConfiguration() {
+  try {
+    let getDataConfiguration = await ubusCall('ns.dedalo', 'get-configuration', {})
```
Comment (andre8244) [inline src/components/standalone/hotspot/StatusContent.vue:35]: See above comments

File: src/components/standalone/hotspot/StatusContent.vue [hunk 26]
```diff
@@ -0,0 +1,212 @@
...
+    if (
+      getDataConfiguration &&
+      getDataConfiguration.data &&
+      getDataConfiguration.data.configuration &&
+      getDataConfiguration.data.configuration.connected
+    )
```
Comment (andre8244) [inline src/components/standalone/hotspot/StatusContent.vue:41]: See above comments

File: src/components/standalone/hotspot/StatusContent.vue [hunk 27]
```diff
@@ -0,0 +1,212 @@
...
+    const res = await ubusCall('ns.dedalo', 'list-sessions')
+    if (res.data && res.data.sessions && res.data.sessions.length)
+      hotspotSession.value = res.data.sessions
+  } catch (err: any) {
+    error.value.notificationTitle = t('error.cannot_load_hotspot_configuration')
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+  }
+}
+
+function secondsToHMS(seconds: number): string {
```
Comment (andre8244) [inline src/components/standalone/hotspot/StatusContent.vue:60]: Consider using `vue-tailwind-lib` function `formatDurationLoc`

File: src/components/standalone/hotspot/StatusContent.vue [hunk 28]
```diff
@@ -0,0 +1,212 @@
...
+
+    const formattedHours = hours.toString().padStart(2, '0')
+    const formattedMinutes = minutes.toString().padStart(2, '0')
+    const formattedSeconds = remainingSeconds.toString().padStart(2, '0')
+
+    return `${formattedHours}:${formattedMinutes}:${formattedSeconds}`
+  } else return '-'
+}
+
+function autoConvertSize(sizeInBytes: number): string {
```
Comment (andre8244) [inline src/components/standalone/hotspot/StatusContent.vue:74]: This function already exists inside `vue-tailwind-lib` and it's called `byteFormat1024`

File: src/components/standalone/hotspot/StatusContent.vue [hunk 29]
```diff
@@ -0,0 +1,212 @@
...
+        </NeButton>
+      </NeEmptyState>
+      <!-- IF CONFIGURATION CONNECTED && EMPTY SESSIIONS -->
+      <NeEmptyState
+        v-if="activeConfiguration && !hotspotSession.length"
+        :title="t('standalone.hotspot.status.empty_sessions')"
+        :description="t('standalone.hotspot.status.empty_sessions_description')"
+        :icon="['fas', 'circle-info']"
+      >
+        <!--NeButton kind="primary" size="lg" @click="emit('goToSetting')">
```
Comment (andre8244) [inline src/components/standalone/hotspot/StatusContent.vue:121]: Let's remove commented code

File: public/i18n/en/translation.json (hunk 5 above)
Comment (andre8244) [reply public/i18n/en/translation.json]: @gsanchietti "... your hotspost service subscription", right?

Comment (gsanchietti) [reply public/i18n/en/translation.json]: Sorry :O

File: public/i18n/en/translation.json [hunk 30]
```diff
@@ -605,6 +615,67 @@
...
+        "login_username": "Username",
+        "login_password": "Password",
+        "login_helper_username": "Enter the email address associated with your Flashstart service subscription.",
+        "configurtion": "Hotspot configuration",
+        "configurtion_unit_name": "Unit name",
+        "configurtion_unit_name_placeholder": "Samplename",
+        "configurtion_unit_description": "Unit description",
+        "configurtion_unit_description_placeholder": "Unit description",
+        "configurtion_network_address": "Network address",
+        "configurtion_network_address_helper": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:655]:
```suggestion
        "configurtion_network_address_helper": "Choose an available network address; devices connected to the hotspot will be assigned an IP address within this network.",
```

File: public/i18n/en/translation.json [hunk 31]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_network_address_placeholder": "192.168.0.0/24",
+        "configurtion_dhcp_range_start": "DHCP range start",
+        "configurtion_dhcp_range_start_helper": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:658]:
```suggestion
        "configurtion_dhcp_range_start_helper": "The initial address within the DHCP range; enter an IP address within the chosen hotspot network. Keep it unchanged if you're uncertain.",
```

File: public/i18n/en/translation.json [hunk 32]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_dhcp_range_start_placeholder": "192.168.0.2",
+        "configurtion_dhcp_range_end": "DHCP range end",
+        "configurtion_dhcp_range_end_helper": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:661]:
```suggestion
        "configurtion_dhcp_range_end_helper": "The last address of the DHCP range; insert an address higher than the one set as the start of the range.",
```

File: public/i18n/en/translation.json [hunk 33]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_dhcp_range_end_placeholder": "192.168.0.254",
+        "configurtion_max_client_allowed": "Max allowed clients:",
+        "configurtion_max_client_allowed_placeholder": "253",
+        "configurtion_parent_hotspot": "Parent hotspot",
+        "configurtion_parent_hotspot_placeholder": "samplename-office",
+        "configurtion_network_device": "Network device",
+        "configurtion_network_device_helper": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:668]:
```suggestion
        "configurtion_network_device_helper": "Choose a network device: the hotspot will intercept all connections on this network interface and enforce authentication for connected clients.",
```

File: public/i18n/en/translation.json [hunk 34]
```diff
@@ -77,6 +81,12 @@
     "cannot_retrieve_protocols": "Cannot retrieve protocols",
     "cannot_retrieve_zones": "Cannot retrieve zones",
     "cannot_retrieve_wan_interfaces": "Cannot retrieve WAN interfaces",
+    "cannot_login_hotspot": "Cannot login",
+    "cannot_login_hotspot_description": "Login failed",
+    "cannot_unregister_hotspot": "Cannot unregister",
+    "cannot_load_hotspot_configuration": "Cannot load hotspot configuration",
+    "empty_network_device": "No network device found.",
```
Comment (andre8244) [inline public/i18n/en/translation.json:88]:
```suggestion
    "empty_network_device": "No network device found",
```

File: public/i18n/en/translation.json [hunk 35]
```diff
@@ -77,6 +81,12 @@
...
+    "empty_network_device_description": "There must be at latest an unassigned device or with role 'hotspot' <br><br> <a href=\"#/standalone/network/interfaces-and-devices\">Go to interfaces and devices</a>",
```
Comment (andre8244) [inline public/i18n/en/translation.json:89]:
```suggestion
    "empty_network_device_description": "There must be at least an unassigned device or a device with role 'hotspot' <br><br> <a href=\"#/standalone/network/interfaces-and-devices\">Go to Interfaces and devices</a>",
```

File: public/i18n/en/translation.json [hunk 36]
```diff
@@ -605,6 +615,67 @@
       "route_onlink": "On-link",
       "route_onlink_description": "When enabled, gateway is on-link even if the gateway does not match any interface prefix"
     },
+    "hotspot": {
+      "title": "Hotspot",
+      "description": "Connect the firewall to a remote hotspot manager instance.",
```
Comment (andre8244) [inline public/i18n/en/translation.json:620]:
```suggestion
      "description": "Connect the unit to a remote hotspot manager instance.",
```

File: public/i18n/en/translation.json [hunk 37]
```diff
@@ -605,6 +615,67 @@
...
+      },
+      "settings": {
+        "login": "Login",
+        "login_description": "Authenticate to edit hotspot configuration or unregister your hotspot.",
+        "login_hostname": "Host name",
+        "login_username": "Username",
+        "login_password": "Password",
+        "login_helper_username": "Enter the username used to access the hotspot manager instance.",
+        "configurtion": "Hotspot configuration",
+        "configurtion_unit_name": "Unit name",
```
Comment (andre8244) [inline public/i18n/en/translation.json:650]:
```suggestion
        "configurtion_unit_name": "Hotspot unit name",
```

File: public/i18n/en/translation.json [hunk 38]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_unit_name_placeholder": "Samplename",
+        "configurtion_unit_description": "Unit description",
```
Comment (andre8244) [inline public/i18n/en/translation.json:652]:
```suggestion
        "configurtion_unit_description": "Hotspot unit description",
```

File: public/i18n/en/translation.json [hunk 39]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_unit_description_placeholder": "Unit description",
```
Comment (andre8244) [inline public/i18n/en/translation.json:653]:
```suggestion
        "configurtion_unit_description_placeholder": "Hotspot unit description",
```

File: public/i18n/en/translation.json [hunk 40]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_network_address": "Network address",
+        "configurtion_network_address_helper": "Choose an available network address; devices connected to the hotspot will be assigned an IP address within this network.",
+        "configurtion_network_address_placeholder": "192.168.0.0/24",
```
Comment (andre8244) [inline public/i18n/en/translation.json:656]: This doesn't need to be translated, you can just put it in the code

File: public/i18n/en/translation.json [hunk 41]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_dhcp_range_start": "DHCP range start",
+        "configurtion_dhcp_range_start_helper": "The initial address within the DHCP range; enter an IP address within the chosen hotspot network. Keep it unchanged if you're uncertain.",
+        "configurtion_dhcp_range_start_placeholder": "192.168.0.2",
```
Comment (andre8244) [inline public/i18n/en/translation.json:659]: Same as above

File: public/i18n/en/translation.json [hunk 42]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_dhcp_range_end": "DHCP range end",
+        "configurtion_dhcp_range_end_helper": "The last address of the DHCP range; insert an address higher than the one set as the start of the range.",
+        "configurtion_dhcp_range_end_placeholder": "192.168.0.254",
```
Comment (andre8244) [inline public/i18n/en/translation.json:662]: Same as above

File: public/i18n/en/translation.json [hunk 43]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_max_client_allowed": "Max allowed clients:",
+        "configurtion_max_client_allowed_placeholder": "253",
```
Comment (andre8244) [inline public/i18n/en/translation.json:664]: No translation needed

File: public/i18n/en/translation.json [hunk 44]
```diff
@@ -605,6 +615,67 @@
...
+        "configurtion_parent_hotspot": "Parent hotspot",
+        "configurtion_parent_hotspot_placeholder": "samplename-office",
+        "configurtion_network_device": "Network device",
+        "configurtion_network_device_helper": "Choose a network device: the hotspot will intercept all connections on this network interface and enforce authentication for connected clients.",
+        "configurtion_network_device_placeholder": "Choose device",
+        "configurtion_use_default_dhcp_range": "Use default DHCP range",
+        "configurtion_save_info": "Save and commit",
+        "configurtion_save_info_description": "All pending changes to the firewall and network configurations will be committed.",
+        "unregister": "Unregister",
+        "unregister_unit": "Unregister unit",
```
Comment (andre8244) [inline public/i18n/en/translation.json:674]:
```suggestion
        "unregister_unit": "Unregister hotspot unit",
```

**Review by andre8244 — APPROVED** [review]

**Pull request merged**